superbench/benchmarks/micro_benchmarks/_export_torch_to_onnx.py

```diff
@@ -138,7 +138,7 @@ def export_torchvision_model(self, model_name, batch_size=1):
             model,
             dummy_input,
             file_name,
-            opset_version=10,
+            opset_version=14,
```

Comment:
are changes in this file needed for this PR on llama2?

Reply:
with the updated transformer package(>=4.28.0), the pytorch models use the scaled_dot_product_attention

else the error in unit testing test_tensorrt_inference_performance.py:

torch.onnx.errors.UnsupportedOperatorError: Exporting the operator 'aten::scaled_dot_product_attention' to ONNX opset version 10 is not supported. Support for this operator was added in version 14, try exporting with this version.